src/classes/RD_RecurringDonations_TEST2.cls

```diff
@@ -1285,7 +986,124 @@ private with sharing class RD_RecurringDonations_TEST2 {
         System.assertEquals(expectedDay, actualDay);
     }
 
-    static testMethod void testOrphanedRd() {
+
+    /*********************************************************************************************************
+    * @description Verifies start date (first Opportunity Close Date)
+    * is on the Day of Month date that is greater or equal to the Date Established.
+    */
+    @isTest
+    private static void nextDonationDateIsGreaterOrEqualToDateEstablished() {
```

Comment:
These four *new* unit tests are written for the PR change. Other changes are made to utilize the common data creation methods and they removed about 300 lines from each recurring donation test class. Let me know if code cleaning changes should be removed.

Reply:
These are great tests @RadSFDeveloper. Do you think there should be a test that does the following: Insert an RD, allowing the Opportunities to be created, then change the NextPaymentDate and save the RD to validate that the changes are as expected?

Reply:
Added `shouldUpdateOpenOppCloseDatesWhenRecDonationNextDonationDateIsChanged()`. Can you please verify the test set up wrt dates is correct as well as no flag is needed to be reset when the recurring donation is updated?